code/modules/paperwork/clipboard.dm

```diff
@@ -1,154 +1,155 @@
+#define PAPERWORK	1
+#define PHOTO		2
+#define IS_PEN(x)	(istype(x, /obj/item/weapon/pen))
+
 /obj/item/weapon/clipboard
 	name = "clipboard"
+	desc = "It looks like you're writing a letter. Want some help?"
 	icon = 'icons/obj/bureaucracy.dmi'
 	icon_state = "clipboard"
 	item_state = "clipboard"
-	throwforce = 0
 	w_class = WEIGHT_CLASS_SMALL
-	throw_speed = 3
-	throw_range = 10
```

Comment:
`throw_range` was removed so it just inherits the default range (which is 7).

`throw_speed` was removed as I played round with the throw speeds and didn't find any noticeable difference between 3 (old), and 2 (which is the value it would inherit). That might be a bug with my repo though...

Should I change the `throw_speed` back?